tests/buyProduct_test.js

```diff
@@ -0,0 +1,50 @@
+const USER = {
+    email: "yulialvju@gmail.com",
+    password: "Use54321!",
+    firstName: "Julia",
+    lastName: "Leonova",
+    phone: "+380932849875",
+    address: "Yesenina, 11",
+    city: "Kharkiv",
+    postcode: "61072",
+}
+
+
+Feature('purchase');
+
+Scenario.only('buy product',  ({ I, homePage }) => {
+   
+    I.login(USER);
+    I.amOnPage('http://opencart.qatestlab.net/index.php?route=product/product&product_id=74');
+    //homePage.clickDropdownCartIcon();
+    //homePage.clickRemoveItems();
+
+    homePage.clickSelectField();
+    homePage.clickColor();
+    homePage.clickAddToCartButton();
+    homePage.clickDropdownCartIcon();
+    homePage.clickCheckout();
+    //homePage.clickStep1Toggle();
+    //I.waitForVisible({xpath: '//input[@id="button-account"]'});
+    //homePage.clickGuestCheckout();
+    //homePage.clickStep1Continue();
+    homePage.fillCheckoutForm2(USER);
+    homePage.clickCountryToggle();
+    //homePage.chooseCountry();
+    //homePage.clickRegionToggle();
+    //homePage.chooseRegion();
+    homePage.clickContinueButton();
+    homePage.clickContinueButton();
+    homePage.clickContinueButton();
+    //I.waitForVisible({xpath: '//label[@for="agree1"]'});
+    homePage.clickContinueButton();
+    homePage.clickAgree();
+    homePage.clickContinueButton();
```

Comment:
you can hide all these Comtinue clicks under single method, e.g. completeSteps1-4

Reply:
added loop

Reply:
> you can hide all these Comtinue clicks under single method, e.g. completeSteps1-4

added loop